stock_inventory_revaluation/tests/test_stock_inventory_revaluation.py

```diff
@@ -234,6 +234,13 @@ def test_inventory_revaluation_price_change_real(self):
                                  'Incorrect inventory revaluation for '
                                  'type Price Change.')
 
+        quants = self.env['stock.quant'].search([('product_id', '=',
+                                                 self.product_real_1.id)])
+
+        # We should be able to delete the quants afterwards
+        for quant in quants:
+            quant.unlink()
```

Comment:
Maybe using .sudo() would solve the issue.

![image](https://cloud.githubusercontent.com/assets/25898232/25431229/fcae10b6-2a7f-11e7-9b2f-f2b06f56ebd9.png)